hummingbird/ml/operator_converters/_base_operator.py

```diff
@@ -16,3 +16,31 @@ def __init__(self, regression=False, classification=False, transformer=False, an
         self.classification = classification
         self.transformer = transformer
         self.anomaly_detection = anomaly_detection
+        self.input_indices = input_indices
+        self.append_output = append_output
+
+    # Helper function to select the input for the current operator
+    def select_input_if_needed(self, x):
+        if self.append_output:
+            self.input_x = x
```

Comment:
what's `input_x`?

Reply:
If this is used from subclasses, maybe we have it as a property with documentation?